CHANGELOG.unreleased.md

```diff
@@ -16,6 +16,7 @@
 - [Home/VM] Homogenize the list of backed up VMs with the normal list (PR [#5046](https://github.com/vatesfr/xen-orchestra/pull/5046)
 - [SR/Disks] Add tooltip for disabled migration (PR [#4884](https://github.com/vatesfr/xen-orchestra/pull/4884))
 - [SR/Advanced, SR selector] Show thin/thick provisioning [#2208](https://github.com/vatesfr/xen-orchestra/issues/2208) (PR [#5081](https://github.com/vatesfr/xen-orchestra/pull/5081))
+- [Licenses] Ability to move a license from another XOA to the current XOA (PR [#5110](https://github.com/vatesfr/xen-orchestra/pull/5110))
```

Comment:
...from the current XOA to an other XOA?

Reply:
No, we want to unbind a license that is attached to an old XOA (maybe deleted or not used any more) and rebind it to the XOA we're connected to. So *from* another XOA *to* this one.